packages/sso/src/oidc/discovery.ts

```diff
@@ -102,27 +107,20 @@ export function computeDiscoveryUrl(issuer: string): string {
  * Future phases will add SSRF protection, allowlist checks, etc.
  *
  * @param url - The discovery URL to validate
```

Comment:
I think we can remove this comment of phase 1 as this pr is adding SSRF protection :)

Reply:
😱 I was pretty sure I had removed these comments, let me fix this.